src/mokernels/moinput.jl

```diff
@@ -24,10 +24,17 @@ The first `out_dim` elements represent all outputs for the first input, the seco
 
 See [Inputs for Multiple Outputs](@ref) in the docs for more info.
 """
-struct MOInputIsotopicByFeatures{S,T<:AbstractVector{S},Tout_dim<:Integer} <:
-       AbstractVector{Tuple{S,Int}}
+struct MOInputIsotopicByFeatures{
+    S,T<:AbstractVector{S},IdxType,ToutIndices<:AbstractVector{IdxType}
+} <: AbstractVector{Tuple{S,IdxType}}
     x::T
-    out_dim::Tout_dim
+    outIndices::ToutIndices
+end
+
+function MOInputIsotopicByFeatures(
+    x::T, out_dim::Tout_dim
+) where {S,T<:AbstractVector{S},Tout_dim<:Integer}
+    return MOInputIsotopicByFeatures{S,T,Int,Base.OneTo{Tout_dim}}(x, Base.OneTo(out_dim))
```

Comment:
Just call the outer constructor:
```suggestion
    return MOInputIsotopicByFeatures(x, Base.OneTo(out_dim))
```